### PR TITLE
Added code changes to show the document information.

### LINKED ISSUE #20: As a developer I want to show the document details, the author and last edited time while the document is opened in the pdf viewer.
Total Days Required 2
Points 2

### PR DESCRIPTION
**Pull request by sireeshanalabothu**
Added code changes to show the document information.
To show author of the document.
Modified date of the document.

### CODE REVIEW

File: src/main/java/edu/wright/airviewer2/AIRViewerController.java [hunk 1]
```diff
@@ -165,20 +183,24 @@ private void refreshUserInterface() {
             addEllipseAnnotationMenuItem.setDisable(false);
             addTextAnnotationMenuItem.setDisable(false);
             deleteAnnotationMenuItem.setDisable(0 >= model.getSelectionSize());
-
+            
+           
+
```
Comment (paidikondala) [inline src/main/java/edu/wright/airviewer2/AIRViewerController.java]: Please format the code

File: src/main/java/edu/wright/airviewer2/AIRViewerController.java [hunk 2]
```diff
@@ -92,7 +102,13 @@ private AIRViewerModel promptLoadModel(String startPath) {
             File file = fileChooser.showOpenDialog(stage);
             if (null != file) {
                 String path = file.getCanonicalPath();
+                while(!path.endsWith(".pdf")) {
+                	System.out.println("select only pdf format files");
+                	file = fileChooser.showOpenDialog(stage);
+                	path = file.getCanonicalPath();
+                }
```
Comment (paidikondala) [inline src/main/java/edu/wright/airviewer2/AIRViewerController.java]: Can you take the latest from master every time when you create PR. I see this issue is already merged to master.

**Review by paidikondala — CHANGES_REQUESTED** [review]
Please add the issue # to PR for traceability

File: src/main/java/edu/wright/airviewer2/AIRViewerController.java [hunk 3]
```diff
@@ -92,7 +102,13 @@ private AIRViewerModel promptLoadModel(String startPath) {
             File file = fileChooser.showOpenDialog(stage);
             if (null != file) {
                 String path = file.getCanonicalPath();
+                while(!path.endsWith(".pdf")) {
```
Comment (aanuhya31) [inline src/main/java/edu/wright/airviewer2/AIRViewerController.java]: Some of the edited code is left uncommented. Could you please check and make the necessary changes?

**Review by aanuhya31 — CHANGES_REQUESTED** [review]

File: src/main/java/edu/wright/airviewer2/AIRViewerController.java (hunk 2 above)
Comment (sireeshanalabothu) [reply src/main/java/edu/wright/airviewer2/AIRViewerController.java]: Sync done with the latest changes

**Comment by sireeshanalabothu** [comment]
by mistake chosen the wrong destination branch

File: src/main/java/edu/wright/airviewer2/AIRViewerController.java (hunk 3 above)
Comment (sireeshanalabothu) [reply src/main/java/edu/wright/airviewer2/AIRViewerController.java]: Given a comment on the requested modified code

**Pull request closed without merging**